app/src/lib/stats/stats-store.ts

```diff
@@ -1006,6 +1012,68 @@ export class StatsStore implements IStatsStore {
     }))
   }
 
+  /**
+   * Increment the number of times the user has opened their external editor
+   * from the suggested next steps view
+   */
+  public async recordSuggestedStepOpenInExternalEditor(): Promise<void> {
```

Comment:
None of these new functions use `await` inside the body, so the `async` can be omitted safely:

```suggestion
  public recordSuggestedStepOpenInExternalEditor(): Promise<void> {
```